R/generate_pseudo_bulk_data.R

```diff
@@ -0,0 +1,350 @@
+#' generate_pseudo_bulk_data 
+#' @param group_by      entry in metadata table, based on these cluster annotation pseudo bulk is performed
+#' @param split_by      variable -> split by a variable within the metadata; k must be a string
+#'                      random -> splits based on a random number; k must be a number
+#'                      Louvain, Louvain_multilevel, SLM, Leiden -> subclusters k must be a list with [resolution, cluster_1, cluster_2]
+#' @param k_variable    variable dependent on the split_by
+#'
+#' @examples
+#' 
+#' #using SCE object
+#' library(scRNAseq)e
```

Comment:
The "e" should be removed.

Question: Did you run build / check correctly? Normally, all examples are executed in this call and this error should be caught during that.